static/js/protein/chart.ts

```diff
@@ -424,7 +545,11 @@ export function drawProteinInteractionChart(
     .attr("transform", "translate(-10,0)rotate(-45)")
     .style("text-anchor", "end");
   // Adds the x-axis text label
-  addXLabel(width, height, "Confidence Score (IntactMiScore)", svg);
+  if (arrayLength < 10) {
```

Comment:
this if statement seems unnecessary since the code in both cases is the exact same?

Reply:
Removed it!